### DISCUSSION TITLE
Hold shift/control to change the height of paths when in free placement mode.

### DISCUSSION THREAD

**Discussion by ocalhoun6** [OpenRCT2/OpenRCT2#17607] [Feature requests] [answered]
I'm talking about the mode that lets you place path tiles with a single click, anywhere on the map -- not the mode that allows you to build paths in a continuous track. That already has this feature, at least for the first placement.

When placing paths in this mode, I don't see any reason why you shouldn't be able to hold down shift or control to change the height of the path you're placing. (Currently, this doesn't happen -- the path gets placed at ground level no matter what.)

Obviously, the click-and-drag smearing effect won't work when holding down shift, so holding down shift would mean you only get one path placement per click. But this could still be useful when you only need a small amount of path, or especially when you're using path as scenery.

Holding down control would be especially useful, though -- you could still hold the mouse button down and smear paths across the map to create elevated walkways quickly and easily. Also useful for things like quickly building a path over a ditch/river, and building over water.

**Comment by X123M3-256** [comment]
> Obviously, the click-and-drag smearing effect won't work when holding down shift, so holding down shift would mean you only get one path placement per click.

It works if you use shift to raise the tile to the desired height, place one tile, then press and hold ctrl and release shift. I know this works because I already implemented this for pools, I don't think it would be difficult to make something similar work for paths.

**Reply by ocalhoun6** [reply]
> raise the tile to the desired height, place one tile, then press and hold ctrl and release shift

Well, yeah. That works ... but only because you've released the shift key before smearing.

If you smeared path while still holding shift, you'd just be smearing paths ... vertically? And I can't think of any case where people would actually want to do that.

> I don't think it would be difficult to make something similar work for paths.

Yeah ... seems like a no-brainer, really. I don't know why this hasn't been thought of before.

**Comment by HtotheTML** [comment]
I don't see why this feature shouldn't be universal across all forms of construction, paths included.

**Comment by sjoerddebruin** [comment, marked as answer]
Will be added with #22569.
